pkg/scheduler/core/generic_scheduler.go

```diff
@@ -740,8 +740,12 @@ func PrioritizeNodes(
 	for i := range priorityConfigs {
 		if priorityConfigs[i].Function != nil {
 			wg.Add(1)
+			metrics.SchedulerGoroutines.WithLabelValues("prioritizing").Inc()
```

Comment:
I wonder if we should break these down to something like "prioritizing-legacy", "prioritizing-mapreduce", "prioritizing-extender", etc.